torch/csrc/jit/tracer.cpp

```diff
@@ -208,7 +216,7 @@ Value* TracingState::getOutput(const IValue& iv, size_t i) {
     return tuple_node->output();
   } else {
     AT_ERROR(
-        "Only tensors or tuples of tensors can be output from traced functions");
+        "Only tensors and list/tuples of tensors can be output from traced functions");
```

Comment:
nit: `Only tensors, lists or tuples of tensors can be output from traced functions`